src/main/java/core/basesyntax/ReturnOperation.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+import java.util.Map;
+
+public class ReturnOperation implements OperationHandler {
+    @Override
+    public void handle(FruitTransaction transaction, Map<String, Integer> storage) {
+        storage.put(transaction.getFruit(), storage.getOrDefault(transaction.getFruit(), 0) + transaction.getQuantity());
```

Comment:
It's good that you're using `Map` methods like `getOrDefault` to simplify the logic. Just ensure that the `storage` map is the appropriate data structure for representing the fruit storage, considering performance and functionality.